### PR TITLE
Add pages for Environment Canada platforms

### PR DESCRIPTION
**Pull request by michaeldavie**
**Description:**
Adds pages for newly created Environment Canada platforms.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):**
home-assistant/home-assistant#15774
home-assistant/home-assistant#15775
home-assistant/home-assistant#15776

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/camera.environment_canada.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ha_category: Weather
+ha_release: 0.74
+ha_iot_class: "Cloud Polling"
+---
+
+The `environment_canada` camera platform uses [Environment Canada](https://weather.gc.ca/mainmenu/weather_menu_e.html) as a source for current meteorological radar imagery.
+
+- The platform automatically determines which radar station to use, according to this hierarchy:
+  - Station code from sensor configuration
+  - Closest station to latitude / longitude from sensor configuration
```
Comment (frenck) [inline source/_components/camera.environment_canada.markdown]: - `Closest` -> `The closest`
- `latitude / longitude` -> `latitude/longitude`

File: source/_components/camera.environment_canada.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  - Closest station to latitude / longitude from Home Assistant core configuration
```
Comment (frenck) [inline source/_components/camera.environment_canada.markdown]: - `Closest` -> `The closest`
- `latitude / longitude` -> `latitude/longitude`

File: source/_components/camera.environment_canada.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+camera:
+  - platform: environment_canada
+    latitude: 50
+    longitude: -100
+    station: XFT
+    name: Ottawa Radar
+    loop: True
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/camera.environment_canada.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/camera.environment_canada.markdown [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+To add Environment Canada radar imagery to your installation, add the desired lines from the following example to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+camera:
+  - platform: environment_canada
+    latitude: 50
+    longitude: -100
+    station: XFT
```
Comment (frenck) [inline source/_components/camera.environment_canada.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/environment_canada.markdown [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+The `environment_canada` component uses [Environment Canada](https://weather.gc.ca/mainmenu/weather_menu_e.html) as a source for current meteorological data.
+
+- This component creates the following entities with their default configurations:
+  - `weather.environment_canada`
+  - `sensor.environment_canada`
+  - `camera.environment_canada` (radar imagery)
+
+To add Environment Canada platforms to your installation, add following to your `configuration.yaml` file:
```
Comment (frenck) [inline source/_components/environment_canada.markdown]: `add following` -> `add the following`

File: source/_components/environment_canada.markdown [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+```yaml
+# Example configuration.yaml entry
+environment_canada:
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/environment_canada.markdown]: Remove this section, or just create a single sentence about not having configuration variables.

File: source/_components/camera.environment_canada.markdown [hunk 7]
```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "Environment Canada Radar"
+description: "Radar imagery from Environment Canada."
+date: 2017-07-07 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: environment_canada.jpg
```
Comment (frenck) [inline source/_components/camera.environment_canada.markdown]: The image does not exist. Please add it to the PR.

File: source/_components/sensor.environment_canada.markdown [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+ha_category: Weather
+ha_release: 0.74
+ha_iot_class: "Cloud Polling"
+---
+
+The `environment_canada` sensor platform uses [Environment Canada](https://weather.gc.ca/mainmenu/weather_menu_e.html) as a source for current meteorological data.
+
+- The sensor automatically determines which weather station to use, according to this hierarchy:
+  - Station code from sensor configuration
+  - Closest station to latitude / longitude from sensor configuration
```
Comment (frenck) [inline source/_components/sensor.environment_canada.markdown]: - `Closest` -> `The closest`
- `latitude / longitude` -> `latitude/longitude`

File: source/_components/sensor.environment_canada.markdown [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+  - Closest station to latitude / longitude from Home Assistant core configuration
```
Comment (frenck) [inline source/_components/sensor.environment_canada.markdown]: - `Closest` -> `The closest`
- `latitude / longitude` -> `latitude/longitude`

File: source/_components/sensor.environment_canada.markdown [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+      - wind_speed
+      - wind_gust
+      - wind_dir
+      - high_temp
+      - low_temp
+      - pop
+      - warning
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/sensor.environment_canada.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/weather.environment_canada.markdown [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+# Example configuration.yaml entry
+weather:
+  - platform: environment_canada
+    latitude: 50
+    longitude: -100
+    station: ON/s0000430
+    name: Ottawa
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/weather.environment_canada.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/weather.environment_canada.markdown [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+ha_category: Weather
+ha_release: 0.74
+ha_iot_class: "Cloud Polling"
+---
+
+The `environment_canada` weather platform uses [Environment Canada](https://weather.gc.ca/mainmenu/weather_menu_e.html) as a source for current meteorological data.
+
+- The platform automatically determines which weather station to use, according to this hierarchy:
+  - Station code from sensor configuration
+  - Closest station to latitude / longitude from sensor configuration
```
Comment (frenck) [inline source/_components/weather.environment_canada.markdown]: - `Closest` -> `The closest`
- `latitude / longitude` -> `latitude/longitude`

File: source/_components/weather.environment_canada.markdown [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+  - Closest station to latitude / longitude from Home Assistant core configuration
```
Comment (frenck) [inline source/_components/weather.environment_canada.markdown]: - `Closest` -> `The closest`
- `latitude / longitude` -> `latitude/longitude`

**Review by frenck — CHANGES_REQUESTED** [review]
@michaeldavie Thank you for your PR!
I've reviewed it and left you some comments. Could you please take a look? 👍

**Comment by michaeldavie** [comment]
Thanks, I've made the changes you pointed out. Please let me know if anything else needs to be adjusted.

**Comment by frenck** [comment]
Thanks for updating the PR @michaeldavie!
Nevertheless, now the build fails.
Could you take a look? 👍

**Comment by michaeldavie** [comment]
My bad; I broke one of the configuration variable descriptions over two lines. The error message was a bit cryptic (it was referring to `Line 12`, but it meant inside the variable block). Anyway, it's back on track.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Nice @michaeldavie!

✅ Approved. Can be merged as soon as the parent PR gets merged in.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed. Feel free to re-open when ready.

**Pull request closed without merging**